src/Employer/UnitTests/Employer.Web/ViewModels/ApplicationReview/ApplicationReviewEditModelTests.cs

```diff
@@ -79,5 +79,21 @@ public void ShouldVeValid(ApplicationReviewStatus outcome, string feedback)
 
             result.IsValid.Should().BeTrue();
         }
+
+        [Theory]
+        [InlineData(ApplicationReviewStatus.Unsuccessful, "?$@#()\"\'\\!,+-=_:;.&€£*%/[] \\A-Z \a-z \0-9 your comments will be sent to the candidate.")]

```

Comment:
As we've only got one set of inline data then this could be just a `[Fact]` test.